packages/nitro-protocol/package.json

```diff
@@ -97,7 +97,7 @@
     "prettier:write": "prettier --write './contracts/**/*.sol'",
     "test": "yarn test:contracts && yarn test:app --all",
     "test:app": "jest -c ./config/jest/jest.config.js",
-    "test:ci": "yarn test:ci:app && yarn test:ci:contracts && yarn test:ci:gas",
+    "test:ci": "yarn test:ci:gas && yarn test:ci:app && yarn test:ci:contracts ",
```

Comment:
A true CircleCI gas optimization.

Any gut feeling about the [fragility / runtime] ratio on `test:ci:app` and `test:ci:contracts`?

Reply:
Hmm I think the contracts probably take longer, but I would need to check.